### PR TITLE
first pass of tags page

### PR DESCRIPTION
**Pull request by enclave-marc-barry**

### CODE REVIEW

File: docs/management/tags.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+For example, if an Administrator creates a `developers` Tag and a `kubernetes-pods` Tag, and then defines a `Kubernetes access` Policy which allows `developers` direct access to `kubernetes-pods`; then systems in the `developers` Tag can change over time, so can members of the `kubernetes-pods` Tag as new instances are spun up and old instances are shutdown. The policy however (the business intent underpinning connectivity between those groups) remains in-place, constant and unchanged.
+
+> **Managed vs. unmanaged systems:** When one or more Tags are attached to a system, that system automatically switches into Managed mode. This means all connectivity for that system must be defined by Policy and that the Enclave software on that system will no longer accept commands from the local user(s). Conversely, if no Tags are attached to a system then the opposite is true; the system enters Unmanaged mode and users are able to configure and control the Enclave software locally.
```
Comment (enclave-alistair) [inline docs/management/tags.md:11]: Suggest "will no longer accept commands from local user(s)" should be clarified to indicate it cannot be modified/configured locally. Users can still give some commands to enclave (stop/start/status, etc).

File: docs/management/tags.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* `f22-raptor.rapid-prototype-team`
+* `nyc-office.developers`
+* `aws-eu-west-2.production.webservers`
+* `dc-lax-11.intranet`
+* `uk.laptops`
+* `uk.laptops.developers`
+* `all-staff`
+* `contractors`
+
+Once defined, account Administrators can add member systems to Tags and composed them policies to create connectivity.
```
Comment (enclave-alistair) [inline docs/management/tags.md:39]:
```suggestion
Once defined, account Administrators can add member systems to Tags and compose Policies to create connectivity.
```

**Review by enclave-alistair — CHANGES_REQUESTED** [review]
Looks pretty good, a couple of minor adjustments.

**Comment by enclave-marc-barry** [comment]
changes made, merging

**Pull request merged**